test/e2e/setup.go

```diff
@@ -154,8 +152,9 @@ func SaveScreenshotAndExit(wd selenium.WebDriver, e error) {
 
 func adminPortalSessionSetup() (string, *selenium.WebDriver) {
 	const (
-		hubPort  = 4444
-		hostPort = 8444
+		hubPort        = 4444
+		hostPort       = 8444
+		KeyVaultPrefix = "KEYVAULT_PREFIX"
```

Comment:
Is this const used?

Reply:
yeah no, also some leftover from another try